inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -277,6 +278,23 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
                 return node->input_value(0).get_partial_shape().rank().get_length() > 5;
             });
 
+    auto normalizeL2FusionCallback = [](const_node_ptr &node) -> bool {
+        auto axes_node = std::dynamic_pointer_cast<const ngraph::opset4::Constant>(node);
+        if (!axes_node)
+            return true;
+        auto axes = axes_node->cast_vector<size_t>();
```

Comment:
We usually do not use size_t to get values from constant due to its variety on platforms. 
```suggestion
        auto axes = axes_node->cast_vector<uin64_t>();
```